errors/getinitialprops-export.md

```diff
@@ -0,0 +1,15 @@
+# getInitialProps Export Warning
+
+#### Why This Warning Occurred
+
+You attempted to statically export your application via `next export`, however, one or more of your pages uses `getInitialProps`.
+
+Next.js discourage you to use `getInitialProps`.
+
+#### Possible Ways to Fix It
+
+Next.js has a proper SSG support, so usage of `next export` should always be paired with `getStaticProps`.
+
+#### Useful Links
+
+- [`getStaticProps` documentation](https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation)
```

Comment:
```suggestion
# getInitialProps Export Warning

#### Why This Warning Occurred

You attempted to statically export your application via `next export`, however, one or more of your pages are using `getInitialProps`.

The use of `getInitialProps` has been discouraged since the release of `getStaticProps`.

#### Possible Ways to Fix It

Next.js has built-in support for Static Generation using `getStaticProps`. Statically exporting your app with `next export` should leverage `getStaticProps` instead.

#### Useful Links

- [`getStaticProps` documentation](https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation)
```